src/cms-content/articles/can-compare.json

```diff
@@ -0,0 +1,7 @@
+}
```

Comment:
Is it intentional that the CAN Analytics links go to Explore in California? Also, not sure if we want to keep that name for the feature